app/src/main/java/com/example/umlandowallet/data/remote/AccessImpl.kt

```diff
@@ -21,12 +21,13 @@ class AccessImpl: Access {
         val relevantTxIds: Array<ByteArray> =
             relevantTxIdsFromChannelManager + relevantTxIdsFromChainMonitor
 
-        this.syncTransactionConfirmed(relevantTxIds, Global.channelManager!!, Global.chainMonitor!!)
         this.syncTransactionsUnconfirmed(
             relevantTxIds,
             Global.channelManager!!,
             Global.chainMonitor!!
         )
+
+        this.syncTransactionConfirmed(relevantTxIds, Global.channelManager!!, Global.chainMonitor!!)
```

Comment:
This is not a comment about this particular line, but more of a general comment: we should try to slowly deal with all the non-null assertion operators that are used in the app. They're perfect for quick iterations and getting stuff to run quickly, but lead to brittle codebases. In general they are regarded as potential danger flags. If a variable is potentially null, we can write code that handles those situations better than just using non-null assertion operators extensively (not a problem in the short term though).

Reply:
Totally agree, still in hacking mode right now!